### PR TITLE
[DA-2146] - Removing whitelist/blacklist verbiage from RDR

### PR DESCRIPTION
**Pull request by dtharpeuno**
## Description of changes/additions
Removed whitelist/blacklist verbiage
** universal config changes will need to follow once on prod

## Tests
- [x] unit tests

### CODE REVIEW

File: rdr_service/alembic/env.py [hunk 1]
```diff
@@ -97,7 +97,7 @@ def run_migrations_offline():
                 context.run_migrations(engine_name=name)
 
 
-autogen_blacklist = set(["aggregate_metrics_ibfk_1"])
+autogen_not_allowed_list = set(["aggregate_metrics_ibfk_1"])
```
Comment (kenny-skaggs) [inline rdr_service/alembic/env.py]: Rather than `not_allowed_list` can we use `denied_list` instead?

File: rdr_service/app_util.py [hunk 2]
```diff
@@ -187,8 +187,7 @@ def get_account_origin_id():
     from rdr_service.api_util import DEV_MAIL
     if not client_id:
         if auth_email == DEV_MAIL:
-            client_id = "example"  # TODO: This is a hack because something sets up configs different
-            # when running all tests and it doesnt have the clientId key.
+            client_id = "example"
```
Comment (kenny-skaggs) [inline rdr_service/app_util.py:192]: Why remove this comment?

File: rdr_service/genomic/genomic_job_components.py [hunk 3]
```diff
@@ -1875,9 +1875,6 @@ def reconcile_metrics_to_data_files(self, genome_type, _gc_site_id):
             file_types_received = set([f.file_type for f in files])
             missing_data_files = required_files_set - file_types_received
 
-            if missing_data_files:
-                self.any_missing_data = True
-
```
Comment (kenny-skaggs) [inline rdr_service/genomic/genomic_job_components.py:1880]: Should there be genomics changes in this PR?

Comment (dtharpeuno) [reply rdr_service/genomic/genomic_job_components.py:1880]: Clean up from proceeding PR, didn't want to open another PR for just this.

File: rdr_service/alembic/env.py (hunk 1 above)
Comment (dtharpeuno) [reply rdr_service/alembic/env.py]: Sure!

File: rdr_service/app_util.py (hunk 2 above)
Comment (dtharpeuno) [reply rdr_service/app_util.py:192]: Reverted

**Review by j-kanuch — APPROVED** [review]

**Review by kenny-skaggs — APPROVED** [review]

**Pull request merged**